libvcx/README.md

```diff
@@ -8,23 +21,20 @@ Layer on top of `aries-vcx` which provides `u32` "handle" reference API. When yo
 create a new object, this layer gives you back `u32` reference to that object, which is how you
 work with it. 
 
-This was historically feasibly approach for building FFI interfaces. Arguably, nowadays
-there's more modern approaches to FFI, for example via [uniffi](https://mozilla.github.io/uniffi-rs/).
-
-### `api_c` module
-Built on top of `api_vcx`, provides runtime executor (tokio) and FFI interface. Java and iOS wrapper
-are linked to this interface.
+This was historically feasibly approach for building FFI interfaces - was used to build 
```

Comment:
"Historically, this was a feasible approach to building...", but I don't think this sentence is necessary as it doesn't add anything to what was already mentioned above.